index.js

```diff
@@ -21,17 +20,31 @@ module.exports = function (mode) {
     ignored: '!!'
   };
 
-  if (mode && !!modeMapping[mode.trim().toLowerCase()]) {
-    mode = modeMapping[mode.trim().toLowerCase()];
-  }
-  mode = (mode || 'M').replace(/[\-\[\]\/\{\}\(\)\*\+\?\.\\\^\$\|]/g, '\\$&');
-  regexTest = new RegExp('^'+mode+'\\s', 'i');
+  if (!_.isArray(modes)) modes = [modes];
+
+  modes = _.chain(modes)
+    .map(function(mode) {
+      var mappedMode;
+      if (!_.isString(mode)) return;
+
+      mappedMode = modeMapping[mode.trim().toLowerCase()]
+      if (!_.isString(mappedMode)) return;
```

Comment:
I see there are no longer tests using the short-hand, and this conditional will cause the API to break, no?
So we'd no longer support using `gitmodified('M')` instead of `gitmodified('modified')`.